packages/deployer/src/cli/commands.ts

```diff
@@ -40,6 +40,14 @@ export async function deploy(
     throw new Error('Invalid mnemonic');
   }
 
+  let apiKey: string | undefined = undefined;
```

Comment:
Rename for a bit more clarity?
```suggestion
  let testingApiKey: string | undefined = undefined;
```